src/gmp/models/filter/convert.js

```diff
@@ -87,6 +87,10 @@ const convert = (keyword, value, relation) => {
     return {value, relation};
   }
 
+  if (isNumberOrNumberString(keyword)) {
+    return {value: `${keyword}${relation}${value}`, relation: '~'};
```

Comment:
After checking how the filter is split in gvmd, adding quotes here will ensure gvmd does not interpret the keyword as a column name
```suggestion
    return {value: `"${keyword}${relation}${value}"`, relation: '~'};
```

Reply:
Thanks for the changes, I've applied them.